opensrp-reveal/src/main/java/org/smartregister/reveal/util/CardDetailsUtil.java

```diff
@@ -55,6 +55,7 @@ public static void formatCardDetails(CardDetails cardDetails) {
             case BusinessStatus.SPRAYED:
             case BusinessStatus.COMPLETE:
             case BusinessStatus.FULLY_RECEIVED:
+            case BusinessStatus.PARTIALLY_SPRAYED:
```

Comment:
Codacy found an issue: [Unnecessary use of fully qualified name 'BusinessStatus.PARTIALLY_SPRAYED' due to existing static import 'org.smartregister.reveal.util.Constants.BusinessStatus.PARTIALLY_SPRAYED'](https://app.codacy.com/manual/OpenSRP/opensrp-client-reveal/pullRequest?prid=5853750)